### PR TITLE
Store the build's commit hash in Jarfile manifest

### PR DESCRIPTION
**Pull request by ghost**
The seednode monitor displays a misleading commit hash, just a hardcoded string which never changes.

This PR fixes that by obtaining the git HEAD commit hash at build time, storing it in the Jarfile manifest information.
Seednodes then report the correct commit hash so that the monitor will indicate what commit they are running.
This is useful to know what source the seednode was built from.

e.g:

image

### CODE REVIEW

File: common/src/main/java/bisq/common/app/Version.java [hunk 1]
```diff
@@ -135,6 +139,16 @@ public static void printVersion() {
                 '}');
     }
 
+    public String readCommitHash() {
```
Comment (HenrikJannsen) [inline common/src/main/java/bisq/common/app/Version.java:142]: Maybe better to use a static method.

File: core/src/main/java/bisq/core/network/p2p/inventory/GetInventoryRequestHandler.java [hunk 2]
```diff
@@ -151,7 +150,7 @@ public void onMessage(NetworkEnvelope networkEnvelope, Connection connection) {
 
             // node
             inventory.put(InventoryItem.version, Version.VERSION);
-            inventory.put(InventoryItem.commitHash, RequestInfo.COMMIT_HASH);
+            inventory.put(InventoryItem.commitHash, new Version().readCommitHash());
```
Comment (HenrikJannsen) [inline core/src/main/java/bisq/core/network/p2p/inventory/GetInventoryRequestHandler.java:153]: A static method would be more natural here.

**Review by HenrikJannsen — APPROVED** [review]
utACK

**Review by alejandrogarcia83 — APPROVED** [review]
utACK

**Pull request merged**